deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapper.java

```diff
@@ -151,6 +151,11 @@ public OpenApiClientGeneratorWrapper withAdditionalModelTypeAnnotationsConfig(fi
         return this;
     }
 
+    public OpenApiClientGeneratorWrapper withRestEasyReactive(Boolean isRestEasyReactive) {
```

Comment:
Not sure if this is a good idea. We should resolve it internally, like you're doing on `OpenApiGeneratorCodeGenBase`, not expose it in the wrapper.

Reply:
Done.